skins/oasis/css/core/ads.scss

```diff
@@ -120,7 +120,7 @@ $video-interactive-area-z-index: $vpaid-z-index + 1;
 
 .wikia-ad {
 	&.slot-animation {
-		@include transition(max-height, 0.8s, ease, 0);
+		@include transition(max-height, 1.6s, ease, 0);
```

Comment:
There was a bug that we set `max-height` on a ad slot to a certain value and it's not changed when page is resized - slot get higher. We had a problem that after resizing video was cut at the bottom because of that `max-height`. We decided to double `max-height` to avoid the problem and keep animation in place. To make this change invisible to end user we had to increase animation time.